lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -114,6 +129,7 @@ def self.process_result(ae_result, aevent)
       MiqPolicy.enforce_policy(target, event_name, inputs) unless target.nil?
     end
   rescue URI::InvalidURIError => err
+    $miq_ae_logger.info(err.message)
```

Comment:
This should log to `error` level instead of `info`.

Reply:
fixed